app/src/test/resources/PhotoResizer.yaml

```diff
@@ -64,8 +64,21 @@ logging:
       maxFileSize: 10MB
 
 # The S3Client configuration
-s3Client:
-  existingBucketName: <your-bucket-name>
-  s3Url: s3.amazonaws.com
-  accessKey: <your-access-key>
-  secretKey: <your-secret-key>
+resizerConfiguration:
+  s3Client:
+    existingBucketName: ${S3_BUCKET:-mra-images}
+    s3Url: ${S3_URL:-http://fake-s3:4569}
+    accessKey: <your-access-key>
```

Comment:
@cstetson Wouldn't be better to have the aws details in a different file like aws_conf.yml and put that in gitignore, with that we will avoid any accidental push of keys.

Reply:
@isserrano updated with env vars